### ISSUE TITLE
domelementtype needs updating to 2.1.0 to avoid typescript errors

### ISSUE THREAD

**Issue by ivnnv** [cheeriojs/dom-serializer#302]
Got this error when migrating a node app that users cheerio to Typescript:

```
Namespace '"/home/.../node_modules/domelementtype/lib/index".ElementType' has no exported member 'Root'.

70 constructor(type: ElementType.Root | ElementType.CDATA | ElementType.Script | ElementType.Style | ElementType.Tag, children: Node[]);
```
This issue has been fixed on `https://github.com/fb55/domelementtype` [already ](https://github.com/fb55/domelementtype/commit/6da046213b2db08d9c1c28fb5f30c7b00d06e9b2), but I had to apply a patch-package in my app to got it solved from now.

And obviously cheerio main package needs to update its dependency from this `dom-serializer` to the new version when this fix is applied.

**Comment by alloystory** [comment]
yes please!

**Comment by fb55** [comment]
Fixed in `1.3.0`.

**Comment by ivnnv** [comment]
You are welcome! 😛
